### DISCUSSION TITLE
Please tell me the first game you played in your life

### DISCUSSION THREAD

**Discussion by hirobf10** [NAIST-SE/AttractingContributors#170] [General]
Hi all!

I want to know the **first** games you played in your life and make a list.
If you have time, please add the game to the list of my README.md and make a Pull Request.

Here is my repository: 
https://github.com/hirobf10/AttractingContributors-FirstGame

Thanks!!

**Comment by ellielo** [comment]
I really don't remember exactly cause it was such a long time ago. But i would think it was probably Super Mario or World of Tanks

**Comment by ElefunAi** [comment]
I have added firstgame list!
Please check and help me!

https://github.com/ElefunAi/AttractingContributors-texteditor

**Reply by hirobf10** [reply]
@ElefunAi 
Thank you!!

I have already sent a Pull Request to your repository and it seems to be merged.
Please check closed Pull Request.

**Comment by veekay7** [comment]
Hey there, I have added my first game I played to your repo.
Please help me out ^^ 
Cheers
https://github.com/VantelinLuca/AttractingContributors-WishCountry
